CONSTITUTION.md

```diff
@@ -108,7 +108,7 @@ on the AtomOne Hub.
 
 Any Cosmonaut can also become a Citizen of the AtomOne Hub by using their
 address to stake ATOMs toward the AtomOne Hub and participate actively in
-governance. The status of citizenship is granted in an autonomous manner.
+governance. The status of citizenship is granted autonomously.
```

Comment:
Do you really believe `autonomously` is  better here? the difference is subtle, but to me `in an autonomous manner` seems more explicit

Reply:
I believe "autonomously" makes more sense here.